jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/VectorImage.java

```diff
@@ -8,6 +8,10 @@
 
 import static com.epam.jdi.light.logger.LogLevels.DEBUG;
 
+/**
+ * To see an example of Vector Image in bootstrap please visit https://getbootstrap.com/docs/4.3/components/card/#example
```

Comment:
Guess this link should point to the same class like in Image element

Reply:
This link is put here because there is only one usage of VectorImage class and it's in CardImageOverlaysTest.java. Using vectorImage the Card Image element is checked on JDI site which has following link - https://getbootstrap.com/docs/4.3/components/card/#image-overlays
So that's the reason why this class description has a link to such but more general element.
But if you feel it will be better to put a link to Images documentation considering above please let me know.